README.md

```diff
@@ -4,71 +4,44 @@ This is the code for OrcPub2.com. Many, many people have expressed interest in h
 
 ## Getting Started with Development
 
-- Install Java: http://openjdk.java.net/ or http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
```

Comment:
I would like to see this modified to have sections, instead of removing the non-docker instructions.

Reply:
I have joined the documentations